jerry-core/ecma/operations/ecma-function-object.c

```diff
@@ -324,6 +324,70 @@ ecma_op_arrow_function_get_compiled_code (ecma_arrow_function_t *arrow_function_
 
 #endif /* !CONFIG_DISABLE_ES2015_ARROW_FUNCTION */
 
+#ifndef CONFIG_DISABLE_ES2015_CLASS
+/**
+ * Helper function for implicit class constructors [[HasInstance]] check.
```

Comment:
Is it possible to add reference to the standard here?

Reply:
I think it's not. This implementation is not part of the standard directly, it's only needed due special implementation of the implicit class constructors.